eng/jacoco-test-coverage/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
Should leave the Azure Core versions as a beta, to verify against `1.2.0` in Maven central.